src/prob/tp_pf.jl

```diff
@@ -36,7 +36,13 @@ function post_tp_pf(pm::GenericPowerModel)
         PMs.variable_dcline_flow(pm, bounded=false, cnd=c)
     end
 
+    add_arcs_trans!(pm)
+    variable_tp_trans_flow(pm, bounded=false)
+
     constraint_tp_voltage(pm)
+    # lower bound is often needed to converge
+    # this constraint only sets bound if previously unset
+    constraint_tp_voltage_mag_unbound(pm, vmin=0.3, vmax=Inf)
```

Comment:
I suggest doing this as a data pre-processing at the top of this formulation.

Reply:
Removed from the PR; unit tests converge without.